homeassistant/components/frontend/www_static/polymer/more-infos/more-info-content.html

```diff
@@ -33,6 +34,7 @@
       dialogOpen: {
         type: Boolean,
         value: false,
+        observer: 'dialogOpenChanged',
```

Comment:
The dialog changed property was not getting passed to child components which is why this observer was added.